modules/openapi/test/resources/foo.smithy

```diff
@@ -28,6 +30,49 @@ operation Greet {
   output: Greeting
 }
 
+@error("client")
+@httpError(404)
+structure NotFound {
+  @required
+  message: String
+}
+
+@examples([
+  {
+      title: "ONE"
+      input: {
+          in: "test input"
+      }
+      output: {
+          out: "test output"
+      }
+  }
+  {
+        title: "TWO"
```

Comment:
Nitpick: Indentation of this one is inconsistent to the indentation of the first.